dockstore-integration-testing/src/test/java/io/dockstore/client/cli/EntryResourceIT.java

```diff
@@ -93,10 +93,40 @@ public void testOrcidExport() {
             Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, e.getCode());
             Assert.assertEquals(EntryResource.VERSION_NOT_BELONG_TO_ENTRY_ERROR_MESSAGE, e.getMessage());
         }
+        try {
+            testingPostgres.runUpdateStatement("insert into token (id, content, refreshToken, tokensource, userid, username, scope) values "
+                + "(9001, 'fakeToken', 'fakeRefreshToken', 'orcid.org', 1, 'Potato', '/authenticate')");
+            entriesApi.exportToORCID(workflowId, null);
+            Assert.fail("Cannot insert the actual scope, must be enum");
+        } catch (ApiException e) {
+            Assert.assertEquals(HttpStatus.SC_INTERNAL_SERVER_ERROR, e.getCode());
+            testingPostgres.runUpdateStatement("delete from token where id=9001");
+        }
+
+        try {
+            testingPostgres.runUpdateStatement("insert into token (id, content, refreshToken, tokensource, userid, username, scope) values "
+                + "(9001, 'fakeToken', 'fakeRefreshToken', 'orcid.org', 1, 'Potato', '/activities/update')");
+            entriesApi.exportToORCID(workflowId, null);
+            Assert.fail("Cannot insert the actual scope, must be enum");
+        } catch (ApiException e) {
+            Assert.assertEquals(HttpStatus.SC_INTERNAL_SERVER_ERROR, e.getCode());
+            testingPostgres.runUpdateStatement("delete from token where id=9001");
+        }
+
         // Give the user a fake ORCID token
-        testingPostgres.runUpdateStatement("insert into token (id, content, refreshToken, tokensource, userid, username) values (9001, 'fakeToken', 'fakeRefreshToken', 'orcid.org', 1, 'Potato')");
+        testingPostgres.runUpdateStatement("insert into token (id, content, refreshToken, tokensource, userid, username, scope) values "
+            + "(9001, 'fakeToken', 'fakeRefreshToken', 'orcid.org', 1, 'Potato', 'AUTHENTICATE')");
```

Comment:
Could emphasize the enum nature of the parameter for future developers with something like `TokenScope.name()`